src/server/analytic_services/prometheus_reporting.js

```diff
@@ -94,6 +125,52 @@ async function start_server(
     }
 }
 
+async function metrics_nsfs_stats_handler() {
+    const nsfs_io_stats = {};
```

Comment:
most of this code is taken from web_server right? can we merge the common logic instead ?

Reply:
@romayalon here the format is in JSON for more readability, but for `web_serve.js` it's in plain HTML format.